packages/terra-clinical-item-collection/src/_ListView.jsx

```diff
@@ -39,7 +39,7 @@ function createListItems(children, onSelect, requiredElements) {
 const ListView = ({
   children, onSelect, isListDivided, requiredElements,
 }) => {
-  const hasAChevron = React.Children.toArray(children).some(child => child.props.showListItemChevron);
+  const hasAChevron = React.Children.toArray(children).some(child => child.props.isSelectable);
```

Comment:
Can we update this const to reflect we are checking for isSelectable to toggle the role assigned below?